.github/workflows/sbt-node-snyk.yml

```diff
@@ -0,0 +1,40 @@
+name: SBT Node Snyk
+
+on:
+  workflow_call:
+    inputs:
+      DEBUG:
+        type: string
+        required: false
+    secrets:
+      SNYK_TOKEN:
+        required: true
+
+jobs:
+    security:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout branch
+              uses: actions/checkout@v2
+
+            - name: Setup debug var
+              run: echo INPUT_DEBUG=${{ inputs.DEBUG }} >> $GITHUB_ENV
+              shell: bash
+
+            - name: Get node version
+              run: echo NODE_VERSION=$(cat .nvmrc) >> $GITHUB_ENV
+
+            - uses: snyk/actions/setup@0.3.0
+            - uses: actions/setup-node@v2
+              with:
+                  node-version: ${{ env.NODE_VERSION }}
+
+            - uses: actions/setup-java@v2
+              with:
+                  java-version: "8"
```

Comment:
You should likely use Java `11`rather than 8

Reply:
Happy to make that change. Out of interest, what affect would you expect this to have on our results from Snyk results / SBT dependencies?

Reply:
I am not certain why we need to setup java in the first place for `snyk monitor`, however if we do we need to set it up, it seems we should use the LTS java version we have moved to.

Reply:
It's a good point, I've made that change now. In regards to the setup, I'll add some more context to the description but essentially using the [Snyk setup action](https://github.com/snyk/actions/tree/master/setup), we are responsible for setting up the environment in which our runner performs the Snyk CLI checks. So in this case we are setting up a Node and Java environment so we can check the dependencies of SBT and Node projects.

Reply:
if there's a quick way to make this configurable (with a default of 11) that would be nice